components/automate-chef-io/Makefile

```diff
@@ -1,38 +1,51 @@
 # we use pushd/popd here, and /bin/sh of our chefes/buildkite image is not bash
 # so we have to override the default shell here
 SHELL=bash
+SWAGGER_RESULT_FILE=static/api-docs/all-apis.swagger.json
+SWAGGER_DIR=data/docs/api_chef_automate
+SWAGGER_FILES = $(shell find $(SWAGGER_DIR) -name '*.swagger.json')
+EXPECTED_CONFLICTS=47
 
 themes/chef:
 	git clone https://${GITHUB_TOKEN}@github.com/chef/chef-hugo-theme.git themes/chef
 
 clean:
 	rm -rf site/themes/chef
+	rm -f $(SWAGGER_RESULT_FILE)
+	rm -rf themes/
+	rm -rf public/
 
 sync: themes/chef
 	pushd themes/chef && git fetch && git reset --hard origin/master && popd
 
-SWAGGER_DIR=data/docs/api_chef_automate
-
 clean_swagger_files:
 	rm -rf ./$(SWAGGER_DIR)/*
 
 sync_swagger_files: clean_swagger_files
 	pushd ../../api/external && \
-		find . -name '*.swagger.json' | sed 's/^\.\///' | while read i ; do \
+		find . -name '*.swagger.json' | while read i ; do \
 			mkdir -p "../../components/automate-chef-io/$(SWAGGER_DIR)/`dirname $$i`" && \
 			cp $$i "../../components/automate-chef-io/$(SWAGGER_DIR)/$$i" \
 		; done && \
 	popd
 
 	pushd ../automate-gateway/api/ && \
-		find . -name '*.swagger.json' | sed 's/^\.\///' | while read i ; do \
-			mkdir -p ../../automate-chef-io/$(SWAGGER_DIR)/`dirname $$i` && \
-			cp $$i ../../automate-chef-io/$(SWAGGER_DIR)/`dirname $$i` \
+		find . -name '*.swagger.json' | while read i ; do \
+			mkdir -p "../../automate-chef-io/$(SWAGGER_DIR)/`dirname $$i`" && \
+			cp $$i "../../automate-chef-io/$(SWAGGER_DIR)/$$i" \
 		; done && \
 	popd
 
-serve: sync
+$(SWAGGER_RESULT_FILE): $(SWAGGER_FILES)
+
+generate_swagger: $(SWAGGER_RESULT_FILE)
+	swagger mixin --quiet -c=$(EXPECTED_CONFLICTS) $(SWAGGER_FILES) > $(SWAGGER_RESULT_FILE)
```

Comment:
This assumes you have swagger installed. As far as I can tell there is no documentation on how to install it currently available.

Reply:
I was just chatting with @vjeffrey about that, I forgot to update the setup instructions. Readme update incoming!

Reply:
Let's add 

```
brew tap go-swagger/go-swagger
brew install go-swagger
```
And a link to https://goswagger.io/install.html#homebrewlinuxbrew for other systems.